application/views/admin/token/invite.php

```diff
@@ -58,10 +58,10 @@
                             <div class='mb-3'>
                                 <?php echo CHtml::label(
                                     gT("Bypass date control before sending email:"),
-                                    'bypassdatecontrol', 
```

Comment:
There are way too many clean-up changes like this, makes it harder to review the PR. Please in the future, do clean-up in separate tasks instead of mixing it up with other. Might also be an automatic setting in your editor you need to check.

Reply:
ok, this was due to my VS code setting,  i will disable.
![image](https://github.com/LimeSurvey/LimeSurvey/assets/86231845/fe810599-74ca-4d12-99aa-91bcb0242a69)

Reply:
i reverted file format changes.